internal/internal_workflow_client.go

```diff
@@ -56,8 +56,8 @@ var (
 )
 
 type (
-	// workflowClient is the client for starting a workflow execution.
-	workflowClient struct {
+	// WorkflowClient is the client for starting a workflow execution.
+	WorkflowClient struct {
```

Comment:
Had to make this public because it is returned by public func `NewServiceClient` and lint complains, if it is private.

Reply:
Yes, I hate this linter check